src/cryptography/__init__.py

```diff
@@ -41,7 +41,7 @@
     )
 if sys.version_info[:2] == (3, 5):
     warnings.warn(
-        "Python 3.5 support will be dropped in the next release of"
+        "Python 3.5 support will be dropped in the next release of "
         "cryptography. Please upgrade your Python.",
```

Comment:
@reaperhulk @alex have you considered including links to issues/PRs in deprecation warnings? I saw some projects doing this and I find it quite useful.

Reply:
That's a good idea -- we've done it in the past once or twice but we should do it for our future support deprecations.